### PR TITLE
chore(server): remove travaux modal and replace with a subpage

### PR DESCRIPTION
**Pull request by euhmeuh**

### CODE REVIEW

File: client/app/dedicated/server/working-status/WORKS.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                <span data-ng-bind="work.details.category"></span>
+            </li>
+            <li>
+                <strong data-i18n-static="modal_working_status_status"></strong>
+                <span data-ng-bind="work.details.status"></span>
+            </li>
+        </ul>
+        <hr>
+    </div>
+</div>
```
Comment (antleblanc) [inline client/app/dedicated/server/working-status/WORKS.html]: `insert_final_newline` please :)

File: client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html [hunk 2]
```diff
@@ -449,14 +449,12 @@
...
                             <div class="oui-tile__definition">
-                                <div data-working-status
-                                     data-modal-id="'worksDedicatedServers'"
-                                     data-product-type="'dedicated_server'"
-                                     data-works-details="works"
-                                     data-show-affiliated="true"
-                                     data-show-only-active="true"
-                                     data-tr="tr">
-                                </div>
+                                <p><span data-i18n-static="core_works_description"></span></p>
```
Comment (antleblanc) [inline client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: Simply:

`<p data-i18n-static="core_works_description"></p>`

File: client/app/dedicated/server/working-status/WORKS.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<div data-ng-controller="WorkingStatusCtrl as $ctrl">
+
+    <button class="oui-button oui-button_link oui-button_icon-left" type="button"
```
Comment (antleblanc) [inline client/app/dedicated/server/working-status/WORKS.html]: `<oui-back-button>` instead?

File: client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    .controller("WorkingStatusCtrl", class WorkingStatusCtrl {
+        constructor ($scope, Products) {
+            this.$scope = $scope;
+            this.Products = Products;
+
+            this.worksDetails = [];
+        }
+
+        $onInit () {
+            this.Products.getWorks("dedicated_server", true, true).then((works) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: `return` statement + `.catch()` maybe?

Comment (euhmeuh) [reply client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: How do you trigger component refresh if the variable already contains a promise?
When the promise resolves, it changes `this.worksDetails` from a promise to the final value, but this does not trigger the Angular binding.

File: client/app/dedicated/server/working-status/WORKS.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<div data-ng-controller="WorkingStatusCtrl as $ctrl">
+
+    <oui-back-button on-click="currentView.value = 'DASHBOARD'"
+                     data-title="{{:: tr('modal_working_status_title') }}">
+    </oui-back-button>
+
+    <div data-ng-repeat="work in $ctrl.worksDetails track by $index">
+        <h5>
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/WORKS.html]: Do you have a `h4`, `h3`, `h2` and `h1` above this ?

File: client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js (hunk 4 above)
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: No loading or anything ?

File: client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html [hunk 6]
```diff
@@ -449,14 +449,12 @@
...
-                                     data-product-type="'dedicated_server'"
-                                     data-works-details="works"
-                                     data-show-affiliated="true"
-                                     data-show-only-active="true"
-                                     data-tr="tr">
-                                </div>
+                                <p data-i18n-static="core_works_description"></p>
+                                <button type="button"
+                                        class="btn btn-block btn-default"
+                                        data-ng-click="currentView.value = 'WORKS'"
```
Comment (jleveugle) [inline client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: Maybe we should improve that and use ui-router?

File: client/app/dedicated/server/working-status/WORKS.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            <a target="_blank"
+               rel="noopener"
+               title="{{ work.title }} ({{ tr('core_new_window') }})"
```
Comment (jleveugle) [inline client/app/dedicated/server/working-status/WORKS.html]: `tr` is not use anymore, you can use angular translate.

So, you can now use `translate-attr` :D

File: client/app/dedicated/server/working-status/WORKS.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+               data-ng-href="{{ work.link }}"
+               data-ng-bind="work.title">
+            </a>
+        </h5>
+        <em data-ng-bind="work.date | date:'short'"></em>
+        <p data-ng-bind-html="work.summary"></p>
+        <ul class="list-inline font-italic">
+            <li>
+                <strong data-i18n-static="modal_working_status_type"></strong>
```
Comment (jleveugle) [inline client/app/dedicated/server/working-status/WORKS.html]: `data-i18n-static` -> `data-translate` :)

File: client/app/dedicated/server/working-status/WORKS.html (hunk 1 above)
Comment (jleveugle) [reply client/app/dedicated/server/working-status/WORKS.html]: and you should configure your editor to do it automatically :D

File: client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js (hunk 4 above)
Comment (jleveugle) [reply client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: @euhmeuh i don't understand your answer :(

We should discuss about that :)

**Comment by antleblanc** [comment]
I close it for inactivity, but feel free to re-open it if needed.

**Comment by antleblanc** [comment]
@euhmeuh You reopened this pull request 6 days ago (with conflicts)

**Comment by euhmeuh** [comment]
@antleblanc I was supposed to work on it and got another priority... I'll close it again.
